plugins/grafana-sankey-plugin/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
As I learned, this seems to be boilerplate code that is automatically generated when creating a grafana plugin.
I don't think we need a sublicense for it, the Antrea license should be good enough